build/mark-new-version.sh

```diff
@@ -92,6 +92,7 @@ fi
 echo "+++ Versioning documentation and examples"
 
 # Update the docs to match this version.
+HTML_PREVIEW_PREFIX="https://htmlpreview.github.io/?https://github.com/GoogleCloudPlatform/kubernetes"
```

Comment:
If you make all the links relative it should Just Work (tm) and then the changes in this file should be unnecessary?